src/transformers/models/layoutlm/modeling_layoutlm.py

```diff
@@ -872,7 +878,52 @@ def forward(
         output_hidden_states=None,
         return_dict=None,
     ):
+        r"""
+        labels (:obj:`torch.LongTensor` of shape :obj:`(batch_size, sequence_length)`, `optional`):
+            Labels for computing the masked language modeling loss. Indices should be in ``[-100, 0, ...,
+            config.vocab_size]`` (see ``input_ids`` docstring) Tokens with indices set to ``-100`` are ignored
+            (masked), the loss is only computed for the tokens with labels in ``[0, ..., config.vocab_size]``
+
+        Returns:
+
+        Examples::
 
+            >>> from transformers import LayoutLMTokenizer, LayoutLMForMaskedLM
+            >>> import torch
+
+            >>> tokenizer = LayoutLMTokenizer.from_pretrained('microsoft/layoutlm-base-uncased')
+            >>> model = LayoutLMForMaskedLM.from_pretrained('microsoft/layoutlm-base-uncased')
+
+            >>> words = ["Hello", "[MASK]"]
+            >>> normalized_word_boxes = [637, 773, 693, 782], [698, 773, 733, 782]
+            >>> inputs = tokenizer(words, return_tensors="pt")
+            >>> input_ids = inputs['input_ids']
+            >>> attention_mask = inputs['attention_mask']
+            >>> token_type_ids = inputs['token_type_ids']
+
+            >>> tokens = []
+            >>> token_boxes = []
+            >>> for word, box in zip(words, normalized_word_boxes):
+            ...    word_tokens = tokenizer.tokenize(word)
```

Comment:
think it should be `>>>` here as well instead of `...` no @sgugger ?

Reply:
For multi-line, it should be `...` after the first line, otherwise the python console interprets each line as standalone.

Reply:
Not sure the indent is right though, it looks like there is a space missing.

Reply:
I copied the code example into a notebook and it runs (to make sure people can directly use it). So not sure if there's anything wrong :)

Reply:
Yeah, there's a space missing for it to be 4-spaces indented, but Python is quite permissive regarding the indentation as long as it's constant. Since it's the only line indented, Python doesn't complain!